example/Douban/test.hs

```diff
@@ -41,9 +39,10 @@ main = do
   print token
   case token of
     Right r -> do
+      -- TODO: display Chinese character. (Text UTF-8 encodeing does not work, why?)
       uid <- authGetBS mgr (accessToken r) [uri|https://api.douban.com/v2/user/~me|]
-      TL.putStrLn $ either TL.decodeUtf8 TL.decodeUtf8 uid
-    Left l -> BSL.putStrLn l
+      putStrLn $ either show (show . TL.decodeUtf8) uid
```

Comment:
Could you help me create a set of keys there? My Chinese is a bit rusty :-). BTW: https://functionalprogramming.slack.com